interfaces/IBF-dashboard/src/assets/i18n/en.json

```diff
@@ -180,8 +180,8 @@
       "active-event": {
         "header": "Trigger: {{ eventName }} in {{firstLeadTimeDate}}",
         "header-below-trigger": "Warning: {{ eventName }} on {{firstLeadTimeDate}}",
-        "header-ongoing": "Ongoing trigger: {{eventName}} in {{firstLeadTimeDate}}",
-        "header-ongoing-below-trigger": "Ongoing warning: {{eventName}} in {{firstLeadTimeDate}}",
+        "header-ongoing": "Ongoing trigger: {{eventName}}",
```

Comment:
During the design review, we were asked to do,

```
"header-ongoing": "Trigger: {{eventName}} ongoing",
"header-ongoing-below-trigger": "Warning: {{eventName}} ongoing",
```

I may be mixing reviews because these changes are in parallel.

Reply:
yeah... this code is being touched in multiple parallel PRs now. This is outdated now and should be trumped by the other change which is in my design-review PR, when hitting merge conflicts on this. In fact, I will make the same change here to already resolve the merge conflict preemptively.

Reply:
done